guardrails/src/drivers/rest/routers/admin.py

```
@@ -0,0 +1,261 @@
+        prompts_yaml=config.prompts_yaml,
+        colang=config.colang,
+        created_at=config.created_at,
+        updated_at=config.updated_at,
+    )
+
+
+@router.get("/configs", response_model=List[ConfigResponse], tags=["configs"])
+async def list_configs(
+    offset: int = Query(0, ge=0),
+    limit: int = Query(100, ge=1, le=1000),
+    uc: ListConfigs = Depends(get_list_configs),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)